examples/QuickJournalSync/AppShell.xaml.cs

```diff
@@ -0,0 +1,13 @@
+﻿using QuickJournalSync.Views;
+
+namespace QuickJournalSync;
+
+public partial class AppShell : Shell
+{
+    public AppShell()
+    {
+        InitializeComponent();
+
+        Routing.RegisterRoute("entryDetail", typeof(EntryDetailPage));
```

Comment:
I know very little about AppShell, but why are the two other pages registered in xaml, and this one is in the code-behind?

Reply:
The pages that are registered here are [global routes](https://learn.microsoft.com/en-us/dotnet/maui/fundamentals/shell/navigation#register-detail-page-routes), they can be theoretically accessed from anywhere in the app. That usually makes sense for "detail" pages, that could be opened from multiple locations.